### PR TITLE
Codecov upgrade

### PR DESCRIPTION
**Pull request by glmapper**

### CODE REVIEW

**Comment by khotyn** [comment]
When this PR will be merged?

**Comment by glmapper** [comment]
> When this PR will be merged?

Yes

**Review by ujjboy — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -51,8 +51,8 @@
         <main.user.dir>${user.dir}</main.user.dir>
         <maven.staging.plugin>1.6.7</maven.staging.plugin>
         <maven.gpg.pluign>1.5</maven.gpg.pluign>
-        <coveralls.maven.plugin>3.1.0</coveralls.maven.plugin>
-        <cobertura.maven.plugin>2.6</cobertura.maven.plugin>
+        <coveralls.maven.plugin>3.2.1</coveralls.maven.plugin>
+        <cobertura.maven.plugin>2.7</cobertura.maven.plugin>
```
Comment (khotyn) [inline pom.xml]: It seems that we are using jacoco to generate the coverage report, do we still need `coveralls` and `cobertura` plugin?

image

**Review by khotyn — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (glmapper) [reply pom.xml]: it could remove those two plugins.

**Review by khotyn — APPROVED** [review]

**Comment by khotyn** [comment]
image
@glmapper Click on the CodeCov report, got the page above, it seems that there is something wrong with our coverage setting.

**Pull request merged**